### PR TITLE
Confirmation to submission

### LINKED ISSUE #758: Rename step 5?
Based off what testing revealed about the confirmation page, maybe we should change its name.

How about we label Step 5 as "Submission" instead?

@keileyo

### PR DESCRIPTION
**Pull request by wpears**
Depends on #764 
Closes #758

### CODE REVIEW

File: src/js/components/Signature.jsx [hunk 1]
```diff
@@ -10,7 +10,7 @@ const showReceipt = ({status, timestamp, receipt, filingPeriod, email}) => {
   if (code !== SIGNED) return null
 
   return (
-    <Alert type="success" heading="HMDA filing completed">
+    <Alert type="success" heading="HMDA filing completed" linebreak={true}>
```
Comment (awolfe76) [inline src/js/components/Signature.jsx]: We can remove the `linebreak={true}`.

File: src/sass/app.scss [hunk 2]
```diff
@@ -14,6 +14,7 @@
 
 // Components
 @import "components/Alert.scss";
+@import "components/SubmissionPageInfo.scss";
```
Comment (awolfe76) [inline src/sass/app.scss]: We aren't using this right? That component takes the [`RefileWarning` styles](https://github.com/cfpb/hmda-platform-ui/blob/0d0cf9fdb65886ba6a215bc8bee54fb8e3138f15/src/js/components/SubmissionPageInfo.jsx#L6).

File: src/sass/components/SubmissionPageInfo.scss [hunk 3]
```diff
@@ -0,0 +1,5 @@
+.SubmissionPageInfo {
```
Comment (awolfe76) [inline src/sass/components/SubmissionPageInfo.scss]: We can get rid of this file.

**Review by awolfe76 — CHANGES_REQUESTED** [review]
Just a couple quick things.

**Comment by wpears** [comment]
Yuck, that stuff came back in when I rebased, removed now.

**Review by awolfe76 — APPROVED** [review]
Nice!

**Pull request merged**